.github/workflows/ci.yml

```
@@ -0,0 +1,27 @@
+name: Node
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 13.x]
```

Let's test on `14.x` instead of `13.x`? I always tend to test on LTS versions only.

```suggestion
        node-version: [10.x, 12.x, 14.x]
```